tethys_cli/cli_helpers.py

```diff
@@ -1,12 +1,25 @@
 import os
 import sys
 import subprocess
+import bcrypt
```

Comment:
Be sure `bcrypt` is included in the environment.yml if it isn't there already. Add other new dependencies too.